nautobot/docs/rest-api/overview.md

```diff
@@ -785,3 +714,309 @@ http://nautobot/api/dcim/sites/ \
 
 !!! note
     The bulk deletion of objects is an all-or-none operation, meaning that if Nautobot fails to delete any of the specified objects (e.g. due a dependency by a related object), the entire operation will be aborted and none of the objects will be deleted.
+
+### Depth Query Parameter
```

Comment:
This should probably be moved earlier in the document, probably in a similar location to the now-removed `Brief Format` section, i.e. adjacent to the list/retrieve object operations since that's what it's most relevant to.